platform-operator/platformcontroller.go

```diff
@@ -847,6 +893,19 @@ func deleteCRDInstances(kind, group, version, plural, namespace string) []byte {
 }
 
 
+func updateCRDInstances(kind, group, version, plural, namespace, chartURL, chartName string) []byte {
+	fmt.Printf("Inside updateCRDInstances...\n")
+	args := fmt.Sprintf("kind=%s&group=%s&version=%s&plural=%s&namespace=%s&chartURL=%s&chartName=%s", kind, group, version, plural, namespace, chartURL, chartName)
```

Comment:
I am wondering if chartURL needs to be URL encoded here as it will contain characters such as "/"?

Reply:
I was wondering that too, and was surprised when it worked without encoding it. I added the URL encoding just now anyways just in case.